cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -1733,7 +1741,7 @@
                   "Fn::Sub": "arn:${AWS::Partition}:s3:::${ResourcesS3Bucket}"
                 },
                 {
-                  "Fn::Sub": "arn:${AWS::Partition}:s3:::${ResourcesS3Bucket}/*"
+                  "Fn::Sub": "arn:${AWS::Partition}:s3:::${ResourcesS3Bucket}/${ArtifactS3RootDirectory}/*"
```

Comment:
I'm afraid that when the customer provide the bucket we need to restrict the policies we ask for..also in the lambda function

Reply:
Let's follow up on this